Convex_hull_3/examples/Convex_hull_3/quickhull_3_SM.cpp

```diff
@@ -0,0 +1,32 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Polyhedron_3.h>
+#include <CGAL/Surface_mesh.h>
+#include <CGAL/convex_hull_3.h>
+#include <vector>
+#include <fstream>
+
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel  K;
+typedef CGAL::Polyhedron_3<K>                     Polyhedron_3;
+typedef K::Point_3                                Point_3;
+typedef CGAL::Surface_mesh<Point_3>               Surface_mesh;
+
```

Comment:
I would like you to try the same example but with `Surface_mesh` instead of `Polyhedron_3`.

I think that call is ambiguous:
```c++
Surface mesh input, output;
CGAL::convex_hull_3(input, output);
```

Reply:
I had a look at the code. I think that call is ambiguous if you add a named parameter argument.

Reply:
I don't understand how the call to `convex_hull_3(input, output, np)` is not ambiguous with `convex_hull_3(begin, end, output)`...

Reply:
oh, it would be 3 times the same type. Not possible with NPs